FirebaseFunctions/Sources/Callable+Codable.swift

```diff
@@ -15,7 +15,12 @@
 import FirebaseSharedSwift
 import Foundation
 
-/// A `Callable` is reference to a particular Callable HTTPS trigger in Cloud Functions.
+/// A `Callable` is reference to a particular Callable HTTPS trigger in Cloud
```

Comment:
nit: Looks like the docs (and code) might have gotten wrapped to 80 chars instead of 100.